cdap-master/src/main/java/co/cask/cdap/data/tools/AbstractUpgrader.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data.tools;
+
+import co.cask.cdap.api.common.Bytes;
+import co.cask.cdap.common.io.Locations;
+import co.cask.cdap.internal.app.ApplicationSpecificationAdapter;
+import com.google.common.base.Throwables;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.apache.twill.filesystem.Location;
+import org.apache.twill.filesystem.LocationFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ * Abstract class for Upgrade
+ */
+public abstract class AbstractUpgrader {
+  private static final Logger LOG = LoggerFactory.getLogger(AbstractUpgrader.class);
+  protected static final byte[] COLUMN = Bytes.toBytes("c");
+  protected static final String DEVELOPER_STRING = "developer";
```

Comment:
Call it ``DEVELOPER_ACCOUNT``? The data type already states that its a ``String``?